libs/embed/src/embed.ts

```diff
@@ -58,6 +61,7 @@ class Novu {
     }
 
     this.clientId = clientId;
+    this.backendUrl = backendUrl;
     this.initializeIframe(clientId, data);
```

Comment:
Add `backendUrl` parameter

`this.initializeIframe(clientId, backendUrl, data);`

Reply:
@remirobichet very good point, I'm also thinking about maybe modifying the interface a bit. Will chat with @AlexVCS or will try to push the updates myself a bit later